terraform/modules/iam_role_policy/aws_broker/policy.json

```diff
@@ -122,6 +122,48 @@
         "*"
       ]
     },
+    {
```

Comment:
Why was this added?

Reply:
I guess we could wait on this but our iam policies that state elasticsearch are going to deprecate. these commands should be the same but with the capability for opensearch.